packages/app/src/cli/services/dev/extension/bundler.ts

```diff
@@ -199,13 +200,20 @@ Redeploy Paths:
       environment: 'development',
       appURL: url,
     })
-      .then(() => {
-        if (!buildSignal.aborted) {
-          return onChange()
-        }
+      .then((reloadedConfig) => {
+        if (buildSignal.aborted) return
+        if (!extension.isAppConfigExtension) return onChange()
```

Comment:
wonder if is it worth it to do this deepCompare for all extensions and not just config ones 🤔 
we already support multiple extensions per toml file and this will help with that

Reply:
Yep, we can do that, it should be as easy as removing line 205 but no need to do it now.